.github/workflows/kind.yaml

```
@@ -0,0 +1,112 @@
+name: Kind
+
+on:
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+env:
+  KIND_VERSION: v0.9.0
+  KIND_CONFIG: .github/kind-config.yaml
+
+jobs:
+  installation-and-connectivitiy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set image tag
+        id: vars
+        run: |
+          if [ ${{ github.event.pull_request.head.sha }} != "" ]; then
+            echo ::set-output name=tag::${{ github.event.pull_request.head.sha }}
+          else
+            echo ::set-output name=tag::${{ github.sha }}
+          fi
+
+      - name: Install Cilium CLI
+        run: |
+          mkdir tmp; cd tmp
+          curl -LO https://github.com/cilium/cilium-cli/releases/download/v0.4/cilium-linux-amd64.tar.gz
+          tar xzvf cilium-linux-amd64.tar.gz
+          sudo mv cilium /usr/bin
+          cd ..; rm -rf tmp
```

[Nitpicking] Maybe we could extract directly to `/usr/bin` for brevity?
```suggestion
          mkdir tmp; cd tmp
          curl -LO https://github.com/cilium/cilium-cli/releases/download/v0.4/cilium-linux-amd64.tar.gz
          sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/bin
          cd ..; rm -rf tmp
```
In which case we might even not need `tmp`:
```suggestion
          curl -LO https://github.com/cilium/cilium-cli/releases/download/v0.4/cilium-linux-amd64.tar.gz
          sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/bin
          rm cilium-linux-amd64.tar.gz
```